modules/cli/src/main/scala/scala/cli/commands/Run.scala

```diff
@@ -72,10 +76,12 @@ object Run extends ScalaCommand[RunOptions] {
         postAction = () => WatchUtil.printWatchMessage(),
         buildTests = false
       ) { res =>
+        processOpt.map(_.destroyForcibly())
```

Comment:
Don't know if we want to destroy the app forcibly here… (Not sure, but I guess this send a SIGKILL to the app.) @armanbilge suggested a Ctrl+C, so a SIGINT, which would be a good default IMO.

@lwronski To send a SIGINT to a process, [this method](https://github.com/VirtusLab/scala-cli/blob/1471648fafe6365d0f138a25019deefa67d8932f/modules/integration/src/test/scala/scala/cli/integration/TestUtil.scala#L116-L120) can be adapted (we need to pass a `-2` to `kill` for Linux / macOS, don't know for Windows). On Linux / macOS, we can even define a GraalVM substitution like done [here](https://github.com/scala-cli/libdaemon-jvm/blob/bd3d8c7d89d4b559505f3b77e579e3135e758cb9/library/src/libdaemonjvm/internal/IsRunningSubst.java), so that from native launchers, we call the `kill` sys call (see `man 2 kill`), rather than calling the external `kill` command (which is `man 1 kill`), which should be faster and more reliable. (You can get the PID of a `java.lang.Process` by calling `pid` on it.)